@here/harp-mapview/lib/text/TextElementsRenderer.ts

```diff
@@ -858,17 +906,15 @@ export class TextElementsRenderer {
 
     private sortTextElements(textElements: TextElement[], maxViewDistance: number) {
         const distancePriorityFactor = 0.1;
-        const indexPriorityFactor = 0.01 * (1 / textElements.length);
+        const uidPriorityFactor = 0.0000000001;
```

Comment:
Why this change from previous formula to hard-coded value?

Reply:
The previous one calculates the priority used the array length, but, the length is changing and this affects the priority across frames. To make the priority value more stable I've used the text element UID. So if the visible tileset is changed during camera movement this does not affect the displayed label's priority.
This value just a very small but static number for reducing the UID weight on text element priority.